x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx

```diff
@@ -265,46 +270,138 @@ export const DefaultFieldRenderer = React.memo(DefaultFieldRendererComponent);
-      data-test-subj="more-container"
-      style={{
-        maxHeight: moreMaxHeight,
-        overflow: 'auto',
-        paddingRight: '2px',
-      }}
-    >
-      {rowItems.slice(overflowIndexStart).map((rowItem, i) => (
-        <EuiText key={`${idPrefix}-${rowItem}-${i}`} size="s">
-          {render ? render(rowItem) : rowItem}
-        </EuiText>
-      ))}
-    </div>
-  )
-);
+  ({
+    attrName,
+    dragDisplayValue,
+    fieldType,
+    idPrefix,
+    isAggregatable,
+    moreMaxHeight,
+    overflowIndexStart,
+    render,
+    rowItems,
+  }) => {
+    const moreItemsWithHoverActions = useMemo(
+      () =>
+        rowItems.slice(overflowIndexStart).reduce<React.ReactElement[]>((acc, rowItem, index) => {
+          const id = escapeDataProviderId(`${idPrefix}-${attrName}-${rowItem}-${index}`);
+          const dataProvider =
+            typeof rowItem === 'string' && attrName != null
+              ? {
+                  and: [],
+                  enabled: true,
+                  id,
+                  name: rowItem,
+                  excluded: false,
+                  kqlQuery: '',
+                  queryMatch: {
+                    field: attrName,
+                    value: rowItem,
+                    displayValue: dragDisplayValue ?? rowItem,
+                    operator: IS_OPERATOR as QueryOperator,
+                  },
+                }
+              : undefined;
+
+          if (!isReputationLink(rowItem) && dataProvider != null) {
+            acc.push(
+              <EuiFlexItem key={`${idPrefix}-${id}`}>
+                <DraggableWrapper
+                  dataProvider={dataProvider}
+                  isDraggable={false}
+                  render={() => (render && render(rowItem)) ?? defaultToEmptyTag(rowItem)}
+                  timelineId={undefined}
+                  fieldType={fieldType}
+                  isAggregatable={isAggregatable}
+                />
+              </EuiFlexItem>
+            );
+          }
+
+          return acc;
+        }, []),
+      [
+        attrName,
+        dragDisplayValue,
+        fieldType,
+        idPrefix,
+        isAggregatable,
+        overflowIndexStart,
+        render,
+        rowItems,
+      ]
+    );
+
+    const moreItems = useMemo(
+      () =>
+        rowItems.slice(overflowIndexStart).map((rowItem, index) => {
+          return (
+            <EuiFlexItem grow={1} key={`${rowItem}-${index}`}>
+              {(render && render(rowItem)) ?? defaultToEmptyTag(rowItem)}
+            </EuiFlexItem>
+          );
+        }),
+      [overflowIndexStart, render, rowItems]
+    );
 
+    return (
+      <div
+        data-test-subj="more-container"
+        style={{
+          maxHeight: moreMaxHeight,
+          overflow: 'auto',
+          paddingRight: '2px',
+        }}
+      >
+        <EuiFlexGroup gutterSize="s" direction="column" data-test-subj="overflow-items">
+          {attrName != null ? moreItemsWithHoverActions : moreItems}
```

Comment:
Hey @angorayc, I couldn't find a unit test for the new property `attrName`. If it doesn't exist, could you add one that checks if the component renders `moreItemsWithHoverActions` when it receives `attrName`?